src/aura/disbursementsCreate/disbursementsCreate.css

```diff
@@ -0,0 +1,21 @@
+.THIS {}
+
+.THIS.wrap {
+    position: relative;
+    display: block;
+    position: relative;
+    width: 100%;
+    min-width: 550px;
+}
+
+.THIS .slds-item_label {
+    width: 50%;
+}
+
+.THIS .slds-item_detail {
+    width: 50%;
```

Comment:
Is there a reason why you're overriding the default label and detail styling?

Reply:
More great feedback. I was increasing the space allotted to the dt and reducing the space given to the dd. I switched to using DL Inline styling to eliminate the need for the overrides.